packages/api/portfolioDrafts/getPortfolioDraft.ts

```diff
@@ -1,18 +1,17 @@
 import { GetCommand } from "@aws-sdk/lib-dynamodb";
 import { APIGatewayProxyEvent, APIGatewayProxyResult } from "aws-lambda";
-import { ErrorCodes } from "../models/Error";
-import { ApiSuccessResponse, ErrorResponse, ErrorStatusCode, SuccessStatusCode } from "../utils/response";
 import { PortfolioDraft } from "../models/PortfolioDraft";
-import { isPathParameterPresent } from "../utils/validation";
 import { dynamodbDocumentClient as client } from "../utils/dynamodb";
 import { DATABASE_ERROR } from "../utils/errors";
+import { ApiSuccessResponse, ErrorResponse, ErrorStatusCode, SuccessStatusCode } from "../utils/response";
+import { isPathParameterPresent } from "../utils/validation";
 
 export const NO_PORTFOLIO_PATH_PARAM = new ErrorResponse(
```

Comment:
Definitely some more cleanup and consolidation to do here but I'll leave it to @zachclark-ccpo to do in 6413 (e.g. moving this and other errors up to a errors.ts).

Reply:
Totally agree. I was trying to fight the urge of scope creeping to fixing _all_ of our error handling stuff. Wanted to try to stay targeted to updating the constructor API and class hierarchy wherever they currently were used (and I think IDE autoformatting betrayed me and caught a few import reorderings as part of that).

Though, there's also likely an argument to make that errors that only have one consumer should not be global/shared errors. `NO_PORTFOLIO_PATH_PARAM` specifically isn't one of those though. But AT-6413 is a good home for those things.